Makefile

```diff
@@ -12,7 +12,7 @@ test: lint testonly
 
 testonly:
 	@echo "✓ Running tests ..."
-	@gotestsum --format pkgname-and-test-fails --no-summary=skipped --raw-command go test -v -json -short -coverprofile=coverage.txt ./...
+	@DATABRICKS_HOST= DATABRICKS_TOKEN= gotestsum --format pkgname-and-test-fails --no-summary=skipped --raw-command go test -v -json -short -coverprofile=coverage.txt ./...
```

Comment:
Please include `DATABRICKS_CONFIG_PROFILE` as well. Pretty common.

Alternatively, this could be a shell script wrapper, e.g. `scripts/cleanenv`, that unsets them in a single spot.

Reply:
+1 to separate shell script wrapper